### ISSUE TITLE
Not working on Android Studio 4.1.1

### ISSUE THREAD

**Issue by ericntd** [badoo/RIBs#223]
When I use File > New > New RIB
Nothing happened for either the 0.8 RIB or the snapshot one

## Versions
Plugin version (from marketplace)
0.3.1-SNAPSHOT-1568815603.957
<img width="423" alt="Screen Shot 2020-11-28 at 11 06 24 AM" src="https://user-images.githubusercontent.com/19919208/100520044-c74d6f80-3169-11eb-80e5-e8f2a7ca2f3d.png">
Framework:
```
buildTools: "29.0.2",
        compileSdk: 29,
        minSdk    : 23,
        targetSdk : 29,
```

## Step to reproduce
I can see the menu item
<img width="691" alt="Screen Shot 2020-11-28 at 11 07 20 AM" src="https://user-images.githubusercontent.com/19919208/100520088-024fa300-316a-11eb-838b-95786aee952e.png">

And the popup
<img width="527" alt="Screen Shot 2020-11-28 at 11 07 29 AM" src="https://user-images.githubusercontent.com/19919208/100520077-f7950e00-3169-11eb-92dc-af4b2b555195.png">

But nothing happend after I entered the name and click "OK"

**Comment by zsoltk** [comment]
Hello @ericntd , can you please give some more details?

- Plugin version
- Framework version
- Did the popup appear to type in the name of the new RIB, or not even that?
- Anything else you think can help to understand this better

Thank you.

**Comment by zsoltk** [comment]
Thanks for the info. This looks like a super outdated version of the plugin. Since we had some issues with marketplace publishing I think we've been building from source for a while. You can do the same inside this project too, and build the `rib-intellij-plugin` module yourself. Let me know how it goes and if it fixes your issue!

**Comment by ericntd** [comment]
Yes, I can build the project but where is the jar file?

When I ran :tooling:rib-intellij-plugin:intellij I got a 0.22.0.jar
Does that look correct?
![Screen Shot 2020-12-08 at 3 00 39 PM](https://user-images.githubusercontent.com/19919208/101535293-70644900-3966-11eb-867a-9a503b2bf117.png)

**Comment by zsoltk** [comment]
Yep, that's the one! If you intend to use it with the latest version, select snapshot templates (should be on the bottom of the dropdown when you try to generate a new one).

Btw once #227 gets merged, it will add 0.23 and 0.24 ones. But again, if you only need the latest, snapshot is the one to pick.
